includes/class-wc-payments.php

```diff
@@ -122,6 +122,9 @@ public static function init() {
 			include_once WCPAY_ABSPATH . 'includes/admin/tracks/tracks-loader.php';
 		}
 
+		// Init Jetpack connection package REST Auth - needs to happen on plugin load, before rest_api_init.
+		\Automattic\Jetpack\Connection\Rest_Authentication::init();
```

Comment:
Is there any problem if this gets executed sooner? I'd prefer if we keep all Jetpack-related initialization in `wcpay_jetpack_init`. That way, if the `add_action( 'plugins_loaded', 'wcpay_jetpack_init', 1 )` filter is removed, no Jetpack-related code will be executed. (For example, in wp.com Simple sites).